code-challenges.test.js

```diff
@@ -40,12 +40,26 @@ describe('arrayContaining', () => {
 
   })
 })
+ // FAIL  ./code-challenges.test.js
+//   ● Test suite failed to run
+// FAIL  ./code-challenges.test.js
+//   arrayContaining
+//     ✕ matches even if received contains additional elements (1 ms)
 
+//   ● arrayContaining › matches even if received contains additional elements
+
+//     ReferenceError: arrayContaining is not defined
+
+//1. create a test
+//2. write a function
+//3.removes first item from array
+//4.randomly shuffles the reast
 
 
 const arrayContaining = (arr)=>{
+    //3.removes first item from array
     var newColors= arr.shift();
```

Comment:
Logic looks excellent!  You don't necessarily need to assign a new variable on line 61.  Looks like you are not using it at any point of your function aside from there.